src/main/kotlin/no/nav/familie/ef/søknad/api/DokumentasjonsbehovController.kt

```diff
@@ -28,4 +31,16 @@ class DokumentasjonsbehovController(private val søknadClient: SøknadClient) {
 
         return ResponseEntity.ok(dokumentasjonsbehovDto)
     }
+    @PostMapping("/person")
+    fun hentDokumentasjonsbehovForPerson(@RequestBody personIdent: PersonIdent): ResponseEntity<List<DokumentasjonsbehovDto>> {
```

Comment:
Tror ikke vi trenger å sende inn personIdent her i body, men kan bruke token direkte
`EksternBrukerUtils.hentFnrFraToken()`

Og kan då endre denne til en GET i stedet for post?

Reply:
Høres lurt ut 👍